Resources/Prototypes/_Goobstation/Entities/Objects/Consumable/Food/lollypops.yml

```
@@ -1264,3 +1268,6 @@
     state: lollypop-wrapper-syndicate
   - type: Item
     sprite: _Goobstation/Objects/Consumable/Food/lollypop.rsi
+  - type: Tag
+    tags: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)